public/js/login.js

```diff
@@ -26,10 +39,19 @@ $(document).ready(() => {
     })
       .then((data) => {
         console.log(data);
-        //window.location.replace("/members");
+        localStorage.setItem("user", JSON.stringify(data));
```

Comment:
if you open two "/login" page and login both user, first of all your localStorage "user" will be messed up by the second user that is logged in

Reply:
Why would we log two users in one localStorage? Isn't local storage client side? I assume it would be ok to override users and only one user has access at the time. Is there something else that's being affected with the local storage?